applications/GeoMechanicsApplication/tests/shell_with_reset_displacement/ProjectParameters.json

```diff
@@ -146,10 +122,10 @@
         "kratos_module": "KratosMultiphysics.GeoMechanicsApplication",
         "process_name":  "ApplyVectorConstraintTableProcess",
         "Parameters":    {
-            "model_part_name": "PorousDomain.YZsupport",
-            "variable_name":   "ROTATION",
+            "model_part_name": "PorousDomain.FixedNodes",
```

Comment:
Now displacements are supported twice. My intention on the 2 clamped lines was: all displacements + 1 rotation around that line.

Reply:
Okay, my intention was to just fix everything for these 3 nodes in the simplest way I knew (also simplifying the visualization). Do you prefer to revert to the YZ/XZ support structure from before?

Reply:
I think "PorousDomain.FixedNodes" is defined twice in the project parameters file now...